applications/openshift/scc/scc_limit_container_allowed_capabilities/rule.yml

```diff
@@ -41,8 +43,11 @@ ocil: |-
     Review each SCC and determine that only required capabilities are either
     completely added as a list entry under <tt>allowedCapabilities</tt>,
     or that all the un-required capabilities are dropped for containers and SCCs.
-
-{{% set jqfilter = '[.items[] | select(.metadata.name | test("{{.var_sccs_with_allowed_capabilities_regex}}"; "") | not)] | map(.allowedCapabilities == null)' %}}
+    varible var_sccs_with_allowed_capabilities_regex can be set to exclude certain
+    SCCs from the check.
+    Use following command to verify if the correct regex is being used, this ouput
+    will list unqualified SCCs:
+    <pre>$ oc get scc -o json | {{{ jqfilter }}}</pre>
```

Comment:
Could we just add a note telling the users to replace the var_sccs_with_allowed_capabilities_regex with the output of the variable to make sure this is really bulletproof?